src/notification/discord.ts

```diff
@@ -0,0 +1,30 @@
+import {Webhook, MessageBuilder} from 'discord-webhook-node';
+import {Config} from '../config';
+import {Logger} from '../logger';
+import {Link} from '../store/model';
+
+const hook = new Webhook(Config.notifications.discord.webHookUrl);
+const notifyGroup = Config.notifications.discord.notifyGroup;
+
+export function sendDiscordMessage(text: string, link: Link) {
```

Comment:
Let's change this to `cartUrl` just to be consistent.

```suggestion
export function sendDiscordMessage(cartUrl: string, link: Link) {
```

Reply:
sure no problem